### PR TITLE
Directory sharing support

### PR DESCRIPTION
**Pull request by edigaryev**
Works both on macOS and Linux.

Requires macOS Ventura.

Resolves https://github.com/cirruslabs/tart/issues/13.

### CODE REVIEW

File: Sources/tart/Commands/Run.swift [hunk 1]
```diff
@@ -37,13 +37,21 @@ struct Run: AsyncParsableCommand {
...
-    """))
+    """, valueName: "path[:ro]"))
   var disk: [String] = []
 
+  @Option(help: ArgumentHelp("""
+                             Additional directory shares with an optional read-only specifier\n(e.g. --dir=\"build:~/src/build\" --dir=\"sources:~/src/sources:ro\")
+                             """, discussion: """
+                                              All shared directories are automatically mounted to "/Volumes/My Shared Files" directory on macOS,
+                                              while on Linux you have to do it manually: "mount -t virtiofs com.apple.virtio-fs.automount /mount/point".
+                                              """, valueName: "name:path[:ro]"))
```
Comment (fkorotkov) [inline Sources/tart/Commands/Run.swift:52]: Why there is a need for a name?

Comment (edigaryev) [reply Sources/tart/Commands/Run.swift:52]: [`VZMultipleDirectoryShare`](https://developer.apple.com/documentation/virtualization/vzmultipledirectoryshare) requires names to guarantee collision-free mounting of multiple directories to a single mountpoint.

File: Sources/tart/VM.swift [hunk 2]
```diff
@@ -278,6 +282,18 @@ class VM: NSObject, VZVirtualMachineDelegate, ObservableObject {
     // Entropy
     configuration.entropyDevices = [VZVirtioEntropyDeviceConfiguration()]
 
+    // Directory share
+    if #available(macOS 13, *) {
```
Comment (fkorotkov) [inline Sources/tart/VM.swift:286]: What do you think throwing an error on Monterey?

Comment (edigaryev) [reply Sources/tart/VM.swift:286]: See https://github.com/cirruslabs/tart/pull/211/commits/01884f7021edb87b6a193c973704dd60950c7c95.

**Review by fkorotkov — APPROVED** [review]

**Pull request merged**